LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-2.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html><!-- webkit-test-runner [ TabsToLinks=true ] -->
```

Comment:
I don't think we should modify test like this. We should fix the test upstream instead.